src/controller/model/endpoint.ts

```diff
@@ -647,7 +647,7 @@ class Endpoint extends Entity {
         } catch (error) {
             error.message = `${log} failed (${error.message})`;
             debug.error(error.message);
-            throw error;
```

Comment:
This should not be removed.

Reply:
I was not aware this was still in the code. It is a remnant of an older test and definitely not wanted.